src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -1835,7 +1835,7 @@ Do you want to install the latest Dynamo update?</value>
     <value>Run started with new working range...</value>
   </data>
   <data name="GroupDefaultText" xml:space="preserve">
-    <value>&lt;Double click here to edit group description&gt;</value>
```

Comment:
Are we using the same resource string in both Core and CoreWpf? Should we get rid of one of them?

Reply:
yes, we are using the same resource string in both, but the Wpf resources are unavailable in Core.